### ISSUE TITLE
Error Couldn't find CamaleonCms::Admin::ApplicationHelper

### ISSUE THREAD

**Issue by buithehien1991** [owen2345/camaleon-cms#901]
Dear author,

I'm using Rails 5.2.0, Ruby 2.5.1 and after install cms, error happen

Couldn't find CamaleonCms::Admin::ApplicationHelper, expected it to be defined in helpers/camaleon_cms/admin/application_helper.rb

I checked this file already on cms library.

Please help me answer it. Thank you

**Comment by alexshenia** [comment]
Same Ruby 2.6.3, Rails 5.1.5

**Comment by alexshenia** [comment]
@buithehien1991 quick fix add to `Gemfile` this row 
```ruby
gem 'cama_contact_form', github: 'alexshenia/cama_contact_form', branch: 'recaptcha-5'
```

**Comment by owen2345** [comment]
Released vs 0.0.29.
